templates/settings.html

```diff
@@ -192,4 +194,71 @@ <h2>Delete Account</h2>
     }
   }
 
+
+  async function deleteAccount(totpToken) {
```

Comment:
Let's comment the JS out completely because `async` is not a thing that IE11 can handle, and will throw parse errors if we leave this available for the code to be executed.

Reply:
Or sorry, I see what happened. Remove the word `async` and the rest can be retained. I'll strike out my comment above.